doc/api/module.md

```diff
@@ -135,7 +135,10 @@ import { findSourceMap, SourceMap } from 'module';
 const { findSourceMap, SourceMap } = require('module');
 ```
 
+<!-- Anchors to make sure old links find a target -->
+<a name="module_module_findsourcemap_path_error"></a>
```

Comment:
`name` attribute is deprecated in HTML5.
```suggestion
<a id="module_module_findsourcemap_path_error"></a>
```

Reply:
I've accidentally let it be known how infrequently I actually write HTML 😆